apps/dashboard/src/app/event/queries.ts

```diff
@@ -13,7 +13,7 @@ interface UseEventAllQueryProps {
 export const useEventAllQuery = ({ filter, page }: UseEventAllQueryProps) => {
   const trpc = useTRPC()
   const { data, ...query } = useQuery({
-    ...trpc.event.all.queryOptions({ filter: { ...filter, includeStatuses: ["DRAFT"] }, ...page }),
+    ...trpc.event.all.queryOptions({ filter: { ...filter, includeStatus: ["DRAFT"] }, ...page }),
```

Comment:
Should be named `byStatus`

Reply:
Wouldn't that be misleading?

Reply:
Or do you want to default to `["PUBLIC"]` and if you give just `["DRAFT"]` you get only drafy

Reply:
Probably yeah, we should probably make some consistent rules for filter naming but I'd like to keep it byStatus to be somewhat consistent until we make further decisions